dockerfiles/theia/Dockerfile

```diff
@@ -26,21 +26,21 @@ ADD supervisord.conf /etc/
 
 RUN \
     # Install needed software
-    apk update && apk add --no-cache make gcc g++ python git bash supervisor jq && \
+    apk update && apk add --no-cache make gcc g++ python git bash supervisor jq openssl && \
     rm -rf /tmp/* /var/cache/apk/* && \
     # Change version of Theia to specified in THEIA_VERSION
     ${HOME}/versions.sh && rm ${HOME}/versions.sh && \
-    # Add Theia extensions
-    #==TODO== replace with master branch after Theia release
-    git clone --depth=1 -b latest-deps https://github.com/eclipse/che-theia-hosted-plugin-manager-extension /tmp/hosted-plugin-extension && \
-    node ${HOME}/add-extensions.js \
-        @eclipse-che/che-theia-hosted-plugin-manager-extension:file:///tmp/hosted-plugin-extension \
-        #che-theia-ssh-extension:https://github.com/eclipse/che-theia-ssh-plugin.git \
-    && rm ${HOME}/add-extensions.js && \
+    # Apply resolution section to the Theia package.json to use strict versions for Theia dependencies
+    node ${HOME}/resolutions-provider.js ${HOME}/package.json && \
+    # generate node_modules, should be reused with default extensions
+    yarn --global-folder ${HOME}/node_modules/ --cache-folder ${HOME}/node_modules/ && \
```

Comment:
how does it behave inside the container when running a new yarn command in for example /tmp/my-project

does yarn will download again all dependencies as cache will be different location ?
(for example if user create a new Theia plugin and want to compile it)

should we use `yarn config set` or `.yarnrc` file

Reply:
@benoitf I think the easiest way to increase docker image size it's a using `yarn cache clean` command. What do you think?